libraries/AP_UAVCAN/AP_UAVCAN.cpp

```diff
@@ -830,6 +862,149 @@ void AP_UAVCAN::notify_state_send()
     _last_notify_state_ms = AP_HAL::native_millis();
 }
 
+void AP_UAVCAN::gnss_send_fix()
+{
+    const AP_GPS &gps = AP::gps();
+
+    if (_gnss.last_gps_lib_fix_ms == gps.last_message_time_ms()) {
```

Comment:
should do "const uint32_t last_ms = gps.last_message_time_ms()" so we don't ask for it twice

Reply:
fixed